examples/vector_tile_raster_2d.html

```diff
@@ -70,7 +70,6 @@
                     name: 'OpenStreetMap',
                     url: 'http://www.openstreetmap.org/',
                 },
-                filter: function (layer) { return ['fill', 'line'].includes(layer.type) },
```

Comment:
could remove the `symbolToCircle` from example ?

Reply:
done